src/nft/components/profile/list/utils.ts

```diff
@@ -183,14 +198,35 @@ export function useUpdateInputAndWarnings(
     const price = listPrice ?? 0
     inputRef.current.value = `${price}`
     if (price < (asset?.floorPrice ?? 0) && price > 0) setWarningType(WarningType.BELOW_FLOOR)
-    else if (asset.floor_sell_order_price && price >= asset.floor_sell_order_price)
+    else if (
+      asset.floor_sell_order_price &&
+      price >= asset.floor_sell_order_price &&
+      asset.asset_contract.tokenType !== NftStandard.Erc1155
+    )
       setWarningType(WarningType.ALREADY_LISTED)
-  }, [asset?.floorPrice, asset.floor_sell_order_price, inputRef, listPrice, setWarningType])
+  }, [
+    asset.asset_contract.tokenType,
+    asset?.floorPrice,
+    asset.floor_sell_order_price,
+    inputRef,
+    listPrice,
+    setWarningType,
+  ])
 }
 
 export const getRoyalty = (listingMarket: ListingMarket, asset: WalletAsset) => {
-  // LooksRare is a unique case where royalties for creators are a flat 0.5% or 50 basis points
-  const baseFee = listingMarket.name === 'LooksRare' ? LOOKS_RARE_CREATOR_BASIS_POINTS : asset.basisPoints ?? 0
+  // LooksRare is a unique case where royalties for creators are a flat 0.5% or 50 basis points if royalty is set
+  const baseFee =
+    listingMarket.name === 'LooksRare'
+      ? asset.basisPoints
+        ? LOOKS_RARE_CREATOR_BASIS_POINTS
+        : 0
+      : asset.basisPoints ?? 0
 
   return baseFee * 0.01
 }
+
+// OpenSea has a 0.5% fee for all assets that do not have a royalty set
+export const getMarketplaceFee = (listingMarket: ListingMarket, asset: WalletAsset) => {
```

Comment:
it might be better to just have an object that stores both the royalties and the basis points for each marketplace that way in a situation where its not just opensea or not just looksrare this doesn't get hectic

Reply:
OS is unique in that they will only charge a market fee if no royalty is set "to fight spam", so there would still need to be unique logic to handle this distinction. 

Current royalty wars are also making so this logic is constantly in flux so I think having these helper fns that we can update quickly as opposed to trying to make long term structural decisions is the best current route until the dust settles.